packages/adapter-begin/index.js

```diff
@@ -24,27 +22,31 @@ function parse_arc(arcPath) {
 	}
 }
 
-export default async function adapter(builder) {
-	builder.log.minor('Parsing app.arc file');
-	const { static: static_mount_point } = parse_arc('app.arc');
-
-	const lambda_directory = resolve(join('src', 'http', 'get-index'));
-	const static_directory = resolve(static_mount_point);
-	const server_directory = resolve(join('src', 'shared'));
-
-	builder.log.minor('Writing client application...');
-	builder.copy_static_files(static_directory);
-	builder.copy_client_files(static_directory);
-
-	builder.log.minor('Building lambda...');
-	const local_lambda_dir = join(__dirname, 'files');
-	copy(local_lambda_dir, lambda_directory);
-
-	builder.log.minor('Writing server application...');
-	builder.copy_server_files(server_directory);
-
-	builder.log.minor('Prerendering static pages...');
-	await builder.prerender({
-		dest: static_directory
-	});
+export default function adapter() {
```

Comment:
I wonder if this should be something like `export default async function createAdapter() {`. I know that Rollup has been pushing the idea that all exports should be named and you should try not to rely on default exports. I don't know if it matters much one way or the other in practice

Reply:
I agree with this sentiment

Reply:
actually i meant to make it an anonymous function. it doesn't really need a name

Reply:
then my comments below can be ignored, but, am I wrong in thinking debugging is easier if anonymous functions are given names?

Reply:
btw since it's inside an adapter anyway, `create` would be a better name I think. Or anonymous if there are no pitfalls.

Reply:
I'm not sure I like `create` because then when you're reading the source where it's called you just see `create()` and it's unclear what you're creating. If we want to keep the name shorter I like `adapter()` more

Reply:
since these are _our_ adapters, we can ensure that nothing throws when we run the anonymous function, so debugging should be a non-issue. (If that fails, debugging still should be fairly simple, since there's no build/sourcemap nonsense.) Other adapter authors are free to name the function whatever they like — it's not part of any API contract or anything so it doesn't matter